lhotse/bin/modes/cut.py

```diff
@@ -45,7 +46,7 @@ def simple(
         cut_set = make_cuts_from_features(feature_set)
     else:
         supervision_set = SupervisionSet.from_yaml(supervision_manifest)
-        cut_set = make_cuts_from_supervisions(feature_set=feature_set, supervision_set=supervision_set)
+        cut_set = make_cuts_from_supervisions_features(feature_set=feature_set, supervision_set=supervision_set)
```

Comment:
that's a step in a good direction - I think we'll eventually group these "make_cuts_" functions into some sort of "Editor"/"Cutter" class that makes them in different ways (not in this PR).